lib/plug/debugger.ex

```diff
@@ -100,17 +100,29 @@ defmodule Plug.Debugger do
qcs7QmvcQqHQ7U0hlrmUwzNffe6EIWAXR/tMInbNBiFCKu6rTEhb3q+/6ZNOj+K2F4+/BZxfURXCfU3yspvx0cAQ4uSvRXRZUow6ePOw4jhLAPLXMSZTO27az+pm7lYP+/JSDZoNsnO0fQpQQm0904ONa/4XgDEEOLRUju/9zOf++TX69g7R/B61u/gOjiQPfVUwcif/WdXfliJvwDTuYTdBQBvsP23M+oI7iD9amHhoEnMt6RGJb5jW2Q37gbKmANaV0nbELfyGhIZ7aFXymyFbQJwCI6961SQZAYZxLhzkTE81LJ24akM4HQuV0daHr9ct5JLYBMBdLnKKdtWK31D0EFPtk4ixTAG3LBjfekzv/CZK99r8OXbrXeU6EBL4v87GJ9kfPWf3D5/ZMc/J8uPMXGYrrWSflLMd26v1LftMP3YoJtCGoGMhg2bslmMDAAuKXxD6EmHekBWqgAzrtdscwA9ghcYZx6AIybDc7RYI30xjMN7sV7uyaI5AAGROxry2mrn/hdM47E0POy0U2KeorhEYpBvzy3n1oQLJ6UM+bigVqEYPRdXzuzufvVP/dyfutd7xLzTBAf+CIgObBNez8i/fP3anw3o07Q0APfZEJwnxfzE7dIecrgZ1MqoTFnYbbqR9AAABslJREFUEJmt21YX366WBRsAIOBW0/iAsVw1RXGyo8K1jfgTGLMZbhtjbgYF9hwBOzOsFhdTw2DMcULHMnfE8dp6/+hZ24l7OMYwuucQYVprGJbDzjFyx0fbATGAGCzTYrIPLAhGMleenpnSgOXrx8/efvmJgyfG7yZN+r2sPxKi9zXP/nzh117/KNN+lsB7wLynckes3JONmBeU8FgwhuC4eB2M/STK96+zrtM0ghoNSrEH24vQXZT6UL2rxlbwH0PA5moBaBy+0f3Nr2xmwvCIhZ9DKfTojakHYSd9fC3O3P0yk0djjsv6Pd9yf1crwN3NMyzcjDYy0m2nbar6DSyCFsvkq5/+3KdfB955/f2g9UdKdOCkgffaPjX8OQGfGICMNjc1UWDKzu2t65NDwxf3Rnz5cBweyb3jMwx72MgzaeMpQAuvFmekjUFxFJDjQjIWU44TQYG+Hdy3jbju2lXMlyOgPcVwweS7kMb+2UynGxYrP35ZZ2//PoBxvbbFzi6wWhn3jFyv1/TBbT1azZgf1zQrsMTaWO/ZSK0GH6ghfbjl9+JbP/PkzxwC3x+CA98HogMbsF6/of/5Ly5/eAg9DvJRJY+dGgObylhB6Q0zmYbf+Km/9YGvTed6Rv47r7yyPH/KT91erM9a2OlwO23i6ZR2PHPQIquChOtxan7TsgzRbfOWoXMAdAUrqvCwjcPDALBJYyaMGpjkavf4BZ659vzxsdMGWyxzqZWtOCwGW62sqkzXRWgauR6NHqP1v8PDBg1+nFpDq8tP/OUnrs2f0feD4MD3ieh9zW9Mz8i/dO31nwL1GI2nqwRIY9XHVgOm3sEowd/+6V9+9Pfe7twHB7Kff/TS7mhHZ4aB55B5IYiH4b5IrcOXDVyZ0jECFdYfeybvFMLOmWxI5ljYusnoWhqwysXxl8f3vfaSHZ1d3LsfvrsHrFbOfSNXtqKZcb020oxD1OQnM+OYbh6jaSHnmmH79sb6t9dXnjh4Yjz5TL5f6/tK9L7mIv9//OPrZ3eW46eC+nFIe6CNJq2C0kDkWKNqlsZ4YX+1899+4u+8524/x1MfB59+HnqrYr0vPPO/H8Jd/2gO6w8AQE+pukHpNCH3NPJhCjskx4SiMHZSVAvbJT1vxt79L+iR16/o3tndLq6Pj49pbhxjYRPh10bbNx4fAfuDW0RYanD3yOPR7trp4+tPPPHEEfCDIXZfPxCiA03kPw2yEexL//jSxXT7BIf8YUucCrMAtDZonRW+3TfyVmb+7p1LX/9q5xRgswGATVOdxz/+uLoR+bv/6sWfIPExGY8DWlK5T+I005YjGN6aA8IDSRLSQlAMzG/ioXtftve+fDi++qHd2A2txjfM/QzNjJ3w5sbO9WOMNg6D7RkZQ+Ti3uLe4dnDW+8GYvf1AyN6XwcHsqdQCRwA+NqvvXbhbuDHbeBHlXqEpKdpnYlDAwdRg5ivjrDnFolvPPYr77//duf/zX/50u4+9FOA/TEh3Zl7BJ3iuPJc+zT9g56t2Q8tr8WQL1z74O98Y//wQ7az3llGhNyd7s5DP6Qf17/NjOtxbWMsbBjWBXcbc73k8v6l+5fuPfnkkyvgzXbND3L9wInel56Rz0X1V/7h5VOrc/gwVvgIPB91y9NBJIJr1pjxsLQrK8tXofUNLf22eLzu59s53N9dL1YXXPwYxAtuGmsoI0dAazKoguoskBxi0EjlDVi8/Mbi5kuXLn7t/g/hh3Z3jnfsxnhDu7u7BgB+WMReLVe2HJe2Xq/N3OjmsRpWq4cPHz789S/++lH3s99NxO7rXUP0vk4SHyjRP+7GBxzDByN10T33AS6SWla1Wx7BcEhqlExBiBXy3W0j6I+msmrS02JBwYyRAu+l80r6+lvL4c6lL37ki/f/5OrccO/eR5br9RUBwN7ent27d4+7u7u2XC3NvcCZq9UqcRrj3bt3Vz9/+PPHc4Diu5HYfb3riN6XnpHjeYgnjLT//g+unvGz4/kd6oIM52V5VuAZAPvToN9W3pTUSDC8wbFSivC4v5DdTcPNWIxXcnF0ffmR/37nk5/85fGff/GfD5+89MnFncM79shDj+jOqTt2+/ZtLhYLDsPAcRy1d38vb9rNODw8HJ9//vnxZOTs3Uzsvt61RO9LEp9+GuyJnJPHn3vmueWdy+/bjYdv7u6tbC9jdw8Acpm7jJGjcLTIpQDcj928d/ru8vjjFy8ezrnymWee8dOnTw97e3vT89jZ2eHx8bEA4OrVq3nx+Yv5eXw+3yo8+m4w0L7T9a4n+nx1LgLeGU7qG6r+enp6vzfD//+FiN/t+r/azlVVWzMEBAAAAABJRU5ErkJggg=="
```

Comment:
If the logo is the same, let's move this from module attributes into regular private functions, so we can share the logo in a single `logo()` function and void duplicating it. WDYT?

Reply:
actually it's different for dark

Reply:
We can't have a single transparent one?

Reply:
it's not transparency, the normal one is just too dark 
![image](https://github.com/user-attachments/assets/b2ec6bcb-0a0e-4b37-8782-286620a5169d)

Reply:
if there was an SVG one, it could be light and dark in the same file theoretically 😃